Lib/test/test_audit.py

```diff
@@ -291,6 +291,20 @@ def test_sys_monitoring_register_callback(self):
 
         self.assertEqual(actual, expected)
 
+    def test_winapi_createnamedpipe(self):
+        winapi = import_helper.import_module("_winapi")
+
+        pipe_name = r"\\.\pipe\LOCAL\test_winapi_createnamed_pipe"
```

Comment:
Can the pipe name contain non-ASCII characters?

Reply:
It can, but the problem being checked here was due to misrepresenting `char *` as `wchar_t *`, and so ASCII is sufficient to turn into corruption when you `s.encode('ascii').decode('utf-16-le')`.